docs/package.json

```
@@ -12,6 +12,7 @@
     "@types/node": "17.0.35",
     "@types/react": "18.0.9",
     "@types/react-dom": "18.0.4",
+    "classnames": "^2.3.1",
```

```suggestion
    "classnames": "2.3.1",
```

Nicer to not have flexible deps. We can add Renovatebot for this repo too